src/main/java/com/example/domain/Employee.java

```diff
@@ -0,0 +1,32 @@
+package com.example.domain;
+
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+
+import lombok.Data;
+
+@Entity
+@Data
+public class Employee {
+
+	@Id
+	@GeneratedValue
+	private Long id;
+	
+	private String firstName;
+	
+	private String lastName;
+	
+	@OneToMany(mappedBy="owner", fetch=FetchType.LAZY)
+	private List<Phone> phones;
+
+	@Override

```

Comment:
By default lombok make deep presentation, including `phones`, but phones may be not in memory. So it is better to specify exactly.